admin/server/billing.go

```diff
@@ -592,6 +597,13 @@ func billingPlanToDTO(plan *billing.Plan) *adminv1.BillingPlan {
 	}
 }
 
+func valOrNullTime(v time.Time) *timestamppb.Timestamp {
```

Comment:
is this mainly to make checks easier at frontend as there won't be any zero check ?

Reply:
Ya it is not standard to do a zero check in UI. It is cleaner to send null. Zero check is a go thing with native support.